express-notes-api/index.js

```diff
@@ -0,0 +1,110 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+app.use(express.json());
+
+const data = require('./data.json');
+const notes = data.notes;
+
+app.get('/api/notes', (req, res) => {
+  if (req.params.id === undefined) {
```

Comment:
There are no path parameters (like ':id') in `/api/notes`, so `req.params.id` will always be undefined.